builders/xcm/xtokens.md

```diff
@@ -0,0 +1,248 @@
+
+!!! note
+    Each parachain can allow/forbid specific methods from a pallet. Consequently, developers must ensure to use methods that are allowed. On the contrary, the transaction will fail with an error similar to `system.CallFiltered`
+
+You'll need `xcUNIT`, which is the [XC-20](/builders/xcm/xc20/) representation of the Alphanet relay chain Token `UNIT`. You can acquire some by swapping for `DEV` tokens (Moonbase Alpha's native token) on [Moonbeam-Swap](https://moonbeam-swap.netlify.app/#/swap), a demo Uniswap-V2 clone on Moonbase Alpha.
+
+![Moonbeam Swap xcUNITs](/images/builders/xcm/xtokens/xtokens-1.png)
+
+To check your `xcUNIT` balance, you can add the XC-20 to MetaMask with the following address: `0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080`. You can check the [XC-20](/builders/xcm/xc20/#calculate-xc20-address) page to learn how to calculate this address.
+
+### X-Tokens Transfer Function {: #xtokens-transfer-function}
+
+In this example, you'll build an XCM message to transfer `xcUNIT` from Moonbase Alpha back to its [relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts)
+
+To transfer assets using the `transfer` function, first head to the extrinsic page of [Polkadot JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/extrinsics). To be able to send an extrinsic, you need to have an [account](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/accounts) with [funds](https://docs.moonbeam.network/builders/get-started/networks/moonbase/#get-tokens) first.
+
+
+Once you have everything set up, set the following options:
+
+1. Select the account from which you want to send the XCM
+2. Choose the **xTokens** pallet
+3. Choose the **trasfer** extrinsic
+4. Set the currency ID to **OtherReserve**. This is because we are not transferring DEV tokens (*SelfReserve*)
+5. Enter the asset ID. For this example, `xcUNIT` has an asset id of `42259045809535163221576417993425387648`
+6. Set the number of tokens to send. For this example, you are sending 1 `xcUNIT`, but you have to account for the 12 decimals
+7. To define the XCM destination multilocation, you have to target an account in the relay chain from Moonbase Alpha as the origin. Therefore, set the following parameters:
+
+    | Parameter |      Value      |
+    |:---------:|:---------------:|
+    |  Version  |      *V1*       |
+    |  Parents  |       *1*       |
+    | Interior  |      *X1*       |
+    |    X1     |  *AccountId32*  |
+    |  Network  |      *Any*      |
+    |    Id     | *TargetAccount* |
+
+8. Set the destination weight to `1000000000`. Note that on Moonbase Alpha, each XCM instruction costs around 100000000 weight units. A `transfer` consists of 4 XCM instructions, so a destination weight of 400000000 should be enough
```

Comment:
I would format the other weight numbers to the be the same format as with the initial `1000000000`